llvm/lib/Transforms/IPO/LowerTypeTests.cpp

```diff
@@ -424,8 +430,10 @@ struct ScopedSaveAliaseesAndUsed {
 class LowerTypeTestsModule {
   Module &M;
 
-  ModuleSummaryIndex *ExportSummary;
-  const ModuleSummaryIndex *ImportSummary;
+  FunctionAnalysisManager &FAM;
+
+  ModuleSummaryIndex *const ExportSummary;
```

Comment:
this was drive-by because I love `const`. Happy to do it in a NFC, but seemed too small and innocuous of a change to bother, but lmk!

Reply:
Please don't